lib/engine/game/g_18_india/game.rb

```diff
@@ -641,17 +646,19 @@ def buyable_bank_owned_companies
         end
 
         def first_bond_in_bank
-          [] << bank.companies.find { |c| c.type == :bond }
+          [bank.companies.find { |c| c.type == :bond }].compact
```

Comment:
This looks a little odd, maybe this instead?

```suggestion
          Array(bank.companies.find { |c| c.type == :bond })
```

Reply:
Will the Array call remove the `nil` result if there isn't a bond?

Reply:
Yep, `Array(nil)` returns `[]`:

```
> docker-compose exec rack bundle exec rake irb
irb -I lib/ -I assets/app/
irb(main):001:0> Array('nil')
=> ["nil"]
irb(main):002:0> Array(nil)
=> []
```